src/UnitTests/GitHub.VisualStudio/UI/Views/GitHubPaneViewModelTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using GitHub.Api;
+using GitHub.Exports;
+using GitHub.Models;
+using GitHub.Primitives;
+using GitHub.Services;
+using GitHub.UI;
+using GitHub.VisualStudio;
+using GitHub.VisualStudio.UI.Views;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Reactive.Linq;
+using UnitTests;
+using Xunit;
+
+public class GitHubPaneViewModelTests : TestBaseClass
+{
+    private readonly IServiceProvider serviceProvider;
+    private readonly IUIController uiController;
+    private readonly FakeMenuCommandService menuCommandService;
+    private readonly GitHubPaneViewModel viewModel;
+    private UIViewType lastUiControllerJump;
+
+    public GitHubPaneViewModelTests()
+    {
+        #region Start tests already logged in with an active repo.
```

Comment:
This is a nit, but I have unreasonable hatred of `#region`s. Could you remove these? It would make me happy :)